src/haddock/gear/prepare_run.py

```diff
@@ -96,7 +115,7 @@ def wrapper(*args, **kwargs):
 
 
 @lru_cache
-def _read_defaults(module_name):
+def _read_defaults(module_name, default_only=True):
```

Comment:
what is default_only? could you write its meaning in the docstring?

Reply:
Good idea !